1008-datagrid-design-guidance.md

```diff
@@ -0,0 +1,34 @@
+---
+category: component design guidance
+state: approved
+created: 2024-10-22
+updated: 2024-10-24
+---
+
+# Datalist Design Guidance
+
+[Datagrid](https://clarity.design/documentation/datagrid) organizes large volumes of data in columns and rows allowing users to scan, compare, and perform other actions.
+
+## Guidance
+
+- **Must** use a datagrid for structured, homogeneous content, where each object has the same attributes.
+- **Must** use a datagrid when users need to search, filter, or perform batch operations on the structured data.
+- **Must not** use to display name/value pairs such as terms and definitions. Consider using a [stack view](https://clarity.design/documentation/stack-view) component instead.
```

Comment:
```suggestion
- Datagrids **must not** be used to display name/value pairs such as terms and definitions. Consider using a [stack view](https://clarity.design/documentation/stack-view) component or a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) instead.
```

We maybe should backlog an issue for description list styling, but I have seen this used in several VCF pages.

Reply:
Feel free to leave the description list part out for now since we don't have Clarity styling or guidance for it yet.